unsupported/test/special_functions.cpp

```diff
@@ -335,6 +335,7 @@ template<typename ArrayType> void array_special_functions()
         ArrayType test = betainc(a, b + one, x) + eps;
         verify_component_wise(test, expected););
   }
+#endif
```

Comment:
this `#endif` has been moved around a bit. is this intended?

Reply:
same reason as above